Modules/IO/NIFTI/test/itkNiftiLargeImageRegionReadTest.cxx

```diff
@@ -0,0 +1,68 @@
+/*=========================================================================
+ *
+ *  Copyright NumFOCUS
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *         https://www.apache.org/licenses/LICENSE-2.0.txt
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *=========================================================================*/
+
+#include "itkImageFileReader.h"
+#include "itkImageFileWriter.h"
+#include "itkTestingMacros.h"
+
+
+int
+itkNiftiLargeImageRegionReadTest(int argc, char * argv[])
+{
+
+  if (argc != 2)
+  {
+    std::cerr << "Missing arguments" << std::endl;
+    std::cerr << "Usage: " << itkNameOfTestExecutableMacro(argv) << " outputFileName" << std::endl;
```

Comment:
Is it necessary to pass the file name as command-line argument?  I'm just wondering, because tests without command-line arguments can be tried out more easily within the IDE (Visual Studio). The file name is always "${ITK_TEST_OUTPUT_DIR}/itkNiftiLargeImageRegionReadTest.nii.gz", right?

Reply:
With a command line argument, the same code could be reused for other testing. As the file is residing in nitfiIO's tests directory, that is less practical than ideal. But a command line argument gives us flexibility of where to put this large output file.

Reply:
> With a command line argument, the same code could be reused for other testing.

Thanks, but is this code really intended to be used for other tests as well? I thought it was specifically for large Nifti files.

Reply:
@N-Dekker I do not like the idea of writing the output data file to the binaries folder for many reasons; I could have required a path, but then it would be to the user to try to guess the filename in case they want to inspect it. Note that as pointed in the related issue, at some point it was necessary to inspect the output file.